src/Maybe.js

```diff
@@ -36,6 +36,18 @@ Maybe.isNothing = function(x) {
   return x === _nothing;
 };
 
+Maybe.toArray = function(m) {
```

Comment:
a separate file would be fine, or we have a grab-bag of utility functions here: https://github.com/ramda/ramda-fantasy/blob/master/src/internal/util.js